tests/testthat/helper.R

```diff
@@ -52,11 +52,6 @@ newDatasetFromFixture <- function(filename) {
     )))
 }
 
-releaseAndReload <- function(dataset) {
-    .releaseDataset(dataset)
-    return(refresh(dataset))
-}
-
```

Comment:
If any of the tests fail, you might need to replace calls to `releaseAndReload` with `refresh`.

Reply:
None fail.